checker/src/org/checkerframework/checker/fenum/FenumAnnotatedTypeFactory.java

```diff
@@ -34,6 +35,7 @@ public FenumAnnotatedTypeFactory(BaseTypeChecker checker) {
         this.postInit();
     }
 
+
```

Comment:
The only change in this file is the added import and this empty line. Is either one necessary?